server/knex/seeds/admin_role.js

```diff
@@ -2,45 +2,91 @@
 // Set us up a default role
 //----------------------------------------------------------------------------------------------------------------------
 
-const logger = require('trivial-logging').loggerFor(module);
-
-//----------------------------------------------------------------------------------------------------------------------
-
 exports.seed = async(knex) =>
 {
-    try
+    // Delete automatic roles
+    await knex('role').del().whereIn('role_id', [ 1, 2 ]);
+
+    // Add Automatic Roles
+    await knex('role')
+        .insert([
+            { role_id: 1, name: 'Admins', permissions: JSON.stringify([ '*/*' ]) },
+            { role_id: 2, name: 'Global Mods', permissions: JSON.stringify([ '*/canModifyContent', '*/canDeleteContent' ]) }
+        ]);
+
+    // Check for admin records, and add them if they're not already there.
+    const admins = [
+        {
+            account_id: 1,
+            hash_id: '3VAAgA',
+            email: 'chris.case@g33xnexus.com',
+            name: 'Morgul',
+            avatar: 'https://lh3.googleusercontent.com/-r8fmbWdlFvg/AAAAAAAAAAI/AAAAAAAAA9g/oWyh8pnmDSY/s96-c/photo.jpg?sz=512',
+            permissions: '[*/*]',
+            settings: '{}',
+            created: knex.fn.now()
+        },
+        {
+            account_id: 2,
+            hash_id: '3VzoXi',
+            email: 'null23544@gmail.com',
+            name: 'Lord Null',
+            avatar: 'https://lh6.googleusercontent.com/-uCa4jsA-_i0/AAAAAAAAAAI/AAAAAAAAAB4/chnr3xPv3_c/s96-c/photo.jpg?sz=512',
+            permissions: '[]',
```

Comment:
Looks like I'm permission-less :P

Reply:
Well, not entirely. Notice how you're in the admin block? You get added to a group with `*/*` permissions.

The only reason I have `*/*` (which is redundant with the admin group) was to enable testing of the permissions system, to make sure that user perms and group perms are additive. I just forgot about that when doing this seed file. ;)